packages/mobile/src/invite/reducer.test.ts

```diff
@@ -1,25 +0,0 @@
-import { storeInviteeData } from 'src/invite/actions'
-import { initialState, inviteReducer as reducer } from 'src/invite/reducer'
-import { mockAccount, mockE164Number } from 'test/values'
-
-describe('invite/reducer', () => {
-  describe(reducer, () => {
-    it('returns the default state', () => {
-      expect(reducer(undefined, {} as any)).toEqual(initialState)
-    })
-  })
-
-  describe(storeInviteeData, () => {
-    it('stores the address-number object', () => {
-      expect(reducer(initialState, storeInviteeData(mockAccount, mockE164Number))).toEqual({
-        isSendingInvite: false,
-        isRedeemingInvite: false,
-        invitees: {
-          [mockAccount]: mockE164Number,
-        },
-        redeemedInviteCode: '',
-        redeemComplete: false,
-      })
-    })
-  })
-})
```

Comment:
I see the `storeInviteeData` action still exists, shouldn't we keep this test?

Reply:
This test was failing and I didn't consider it work fixing because reducers are so trivially simple.